docs/running-notebooks/what-is-session/index.md

```diff
@@ -1,5 +1,10 @@
 # What is a session?
 
-Alongside running batch computational workflows, the REANA platform allows users to open interactive sessions (such as [Jupyter notebooks based](../opening-sessions#jupyter-notebooks-based-sessions)) on the same workspace.
+Alongside running batch computational workflows, the REANA platform
+allows users to open interactive sessions (such as
+[Jupyter notebooks](../opening-sessions#jupyter-notebooks-based-sessions))
+based on the same workspace.
 
-The opened sessions will have access to the workflow workspace, allowing users to modify the analysis and its assets and quickly debug common problems.
+The opened sessions will have access to the workflow workspace, allowing
+users to modify the analysis and its assets and quickly debug common
```

Comment:
allowing users to modify the analysis and its assets, to run interactive code to explore the produced data on the workspace, or to open a terminal session on the workflow workspace in order to debug problems.